keep/README.md

```diff
@@ -0,0 +1,57 @@
+# Agent Check: Keep
+
+## Overview
+
+Keep is an open-source AIOps alert management tool.
+
+Keep acts as a single pane of glass and helps you turn 1000s into just 10s of meaningful alerts.
+
+We use monitors data, logs and events to correlate alerts and reduce noise.
```

Comment:
Does this integration send data into Datadog at all? If so, can you explain the types of data/value of that data as well?

Reply:
Hey @anawishnoff :) We plan on making it send our monitors "fatiguness meter" metric very soon. I'll update the documentation and the integration as soon as we do.